style.css

```diff
@@ -0,0 +1,3 @@
+﻿h1 {
+    color: #0094ff;
+}
```

Comment:
Please let's fix the `Stylint` and `Webhint` errors, the purpose of Linting is to check our source code for programmatic and stylistic errors. 💪

![image](https://github.com/Movgto/Hello_Microverse/assets/78251927/ec874a79-cb09-4329-b5bf-c7bbf8a74256)

![image](https://github.com/Movgto/Hello_Microverse/assets/78251927/0ffcf6a6-bf73-4195-9767-200146f22a99)

Kindly read through the errors to fix them. 🤗🛠